libnmstate/nm/profile.py

```diff
@@ -456,6 +458,12 @@ def _generate_connection_settings(self, ifname):
                 OVSInterface.PATCH_CONFIG_SUBTREE
             )
             settings.extend(nm_ovs.create_interface_setting(patch_state))
+        elif self.iface.type == InterfaceType.INFINIBAND:
+            ib_setting = create_infiniband_setting(
```

Comment:
Same here. I prefer importing only the function. Will refactor the others in another PR.

Reply:
Yes. I prefer import function only.